.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Falco Publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install -U pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
where these variable will be stored ? can I publish the wheel package ?

Reply:
Also I think we should publish falco-0.1.0.tar.gz in github releases https://github.com/falcosecurity/client-py/releases ,again I am open to better ways to do it.

Reply:
The publish step gets triggered with a manual release